extensions/amp-connatix-player/0.1/test/validator-amp-connatix-player.out

```diff
@@ -28,6 +28,15 @@ amp-connatix-player/0.1/test/validator-amp-connatix-player.html:23:2 The mandato
 |      width="150"
 |      height="80">
 |    </amp-connatix-player>
+|    <!-- Invalid: dock without amp-video-docking extension -->
+|    <amp-connatix-player
+>>   ^~~~~~~~~
+amp-connatix-player/0.1/test/validator-amp-connatix-player.html:29:2 The attribute 'dock' requires including the 'amp-video-docking' extension JavaScript.
+|      layout="responsive"
```

Comment:
missing `data-player-id` here. 
I'm still looking into the other error, hoping to figure it out by today

Reply:
can you copy this file exactly https://github.com/ampproject/amphtml/pull/39601/files#diff-632c252f906759e11a7fc6bbc1d08f6832d619b9ac86f61926535c8ea22dad7c

theres a newline required on the end i think.